### PR TITLE
Rename fc_size to output_size

### PR DESCRIPTION
**Pull request by ShreyaR**

### CODE REVIEW

File: ludwig/modules/attention_modules.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def output_shape(self):
 
 
 class TransformerBlock(LudwigModule):
-    def __init__(self, input_size, sequence_size, hidden_size, num_heads, fc_size, dropout=0.1):
+    def __init__(self, input_size, sequence_size, hidden_size, num_heads, output_size, dropout=0.1):
```
Comment (w4nderlust) [inline ludwig/modules/attention_modules.py:91]: add type hints?

File: ludwig/modules/attention_modules.py [hunk 2]
```diff
@@ -123,7 +123,15 @@ def output_shape(self) -> torch.Size:
 
 class TransformerStack(LudwigModule):
     def __init__(
-        self, input_size, sequence_size, hidden_size=256, num_heads=8, fc_size=256, num_layers=1, dropout=0.1, **kwargs
+        self,
```
Comment (w4nderlust) [inline ludwig/modules/attention_modules.py:126]: add type hints?

File: tests/integration_tests/test_audio_feature.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 BATCH_SIZE = 2
 SEQ_SIZE = 20
 AUDIO_W_SIZE = 16
-DEFAULT_FC_SIZE = 256
+DEFAULT_OUTPUT_SIZE = 256
```
Comment (w4nderlust) [inline tests/integration_tests/test_audio_feature.py:10]: this default is likely too big, let's decrease it for faster tests

File: tests/integration_tests/test_hyperopt.py [hunk 4]
```diff
@@ -48,7 +48,7 @@
         },
         "combiner.fc_layers": {
             "type": "category",
-            "values": [[{"fc_size": 512}, {"fc_size": 256}], [{"fc_size": 512}], [{"fc_size": 256}]],
+            "values": [[{"output_size": 512}, {"output_size": 256}], [{"output_size": 512}], [{"output_size": 256}]],
```
Comment (w4nderlust) [inline tests/integration_tests/test_hyperopt.py:51]: this default is likely too big, let's decrease it for faster tests

File: tests/integration_tests/test_hyperopt.py [hunk 5]
```diff
@@ -158,9 +158,13 @@ def test_hyperopt_run_hyperopt(csv_filename, samplers):
             output_feature_name
             + ".fc_layers": {
                 "type": "category",
-                "values": [[{"fc_size": 512}, {"fc_size": 256}], [{"fc_size": 512}], [{"fc_size": 256}]],
+                "values": [
+                    [{"output_size": 512}, {"output_size": 256}],
+                    [{"output_size": 512}],
+                    [{"output_size": 256}],
+                ],
```
Comment (w4nderlust) [inline tests/integration_tests/test_hyperopt.py:165]: these values are likely too big, let's decrease it for faster tests

File: tests/integration_tests/test_hyperopt.py [hunk 6]
```diff
@@ -158,9 +158,13 @@ def test_hyperopt_run_hyperopt(csv_filename, samplers):
...
             },
-            output_feature_name + ".fc_size": {"type": "int", "low": 32, "high": 256, "steps": 5},
+            output_feature_name + ".output_size": {"type": "int", "low": 32, "high": 256, "steps": 5},
```
Comment (w4nderlust) [inline tests/integration_tests/test_hyperopt.py:167]: these values is likely too big, let's decrease it for faster tests

File: tests/integration_tests/test_hyperopt_ray.py [hunk 7]
```diff
@@ -47,9 +47,9 @@
         "utterance.fc_layers": {
             "space": "choice",
             "categories": [
-                [{"fc_size": 512}, {"fc_size": 256}],
-                [{"fc_size": 512}],
-                [{"fc_size": 256}],
+                [{"output_size": 512}, {"output_size": 256}],
+                [{"output_size": 512}],
+                [{"output_size": 256}],
```
Comment (w4nderlust) [inline tests/integration_tests/test_hyperopt_ray.py:52]: these values is likely too big, let's decrease it for faster tests

File: tests/integration_tests/test_hyperopt_ray.py [hunk 8]
```diff
@@ -204,7 +204,7 @@ def test_hyperopt_run_hyperopt(csv_filename):
                     "lower": 0.001,
                     "upper": 0.1,
                 },
-                output_feature_name + ".fc_size": {"space": "randint", "lower": 32, "upper": 256},
+                output_feature_name + ".output_size": {"space": "randint", "lower": 32, "upper": 256},
```
Comment (w4nderlust) [inline tests/integration_tests/test_hyperopt_ray.py:207]: these values is likely too big, let's decrease it for faster tests

File: tests/integration_tests/test_hyperopt_ray_horovod.py [hunk 9]
```diff
@@ -277,7 +277,7 @@ def test_hyperopt_run_hyperopt(csv_filename, ray_mock_dir):
                 "lower": 0.001,
                 "upper": 0.1,
             },
-            output_feature_name + ".fc_size": {"space": "randint", "lower": 32, "upper": 256},
+            output_feature_name + ".output_size": {"space": "randint", "lower": 32, "upper": 256},
```
Comment (w4nderlust) [inline tests/integration_tests/test_hyperopt_ray_horovod.py:280]: these values is likely too big, let's decrease it for faster tests

File: tests/integration_tests/utils.py [hunk 10]
```diff
@@ -223,7 +223,7 @@ def audio_feature(folder, **kwargs):
             {"filter_size": 400, "pool_size": 16, "num_filters": 32, "regularize": "false"},
             {"filter_size": 40, "pool_size": 10, "num_filters": 64, "regularize": "false"},
         ],
-        "fc_size": 256,
+        "output_size": 256,
```
Comment (w4nderlust) [inline tests/integration_tests/utils.py:226]: these values is likely too big, let's decrease it for faster tests

File: tests/ludwig/features/test_image_feature.py [hunk 11]
```diff
@@ -36,7 +36,7 @@ def image_config():
         "pool_strides": None,
         "fc_layers": None,
         "num_fc_layers": 1,
-        "fc_size": 256,
+        "output_size": 256,
```
Comment (w4nderlust) [inline tests/ludwig/features/test_image_feature.py:39]: these values is likely too big, let's decrease it for faster tests

File: ludwig/modules/attention_modules.py (hunk 1 above)
Comment (ShreyaR) [reply ludwig/modules/attention_modules.py]: Done

File: ludwig/modules/attention_modules.py (hunk 2 above)
Comment (ShreyaR) [reply ludwig/modules/attention_modules.py:134]: Done

File: tests/integration_tests/test_audio_feature.py (hunk 3 above)
Comment (ShreyaR) [reply tests/integration_tests/test_audio_feature.py:10]: Done

File: tests/integration_tests/test_hyperopt.py (hunk 4 above)
Comment (ShreyaR) [reply tests/integration_tests/test_hyperopt.py]: Done

File: tests/integration_tests/test_hyperopt.py (hunk 5 above)
Comment (ShreyaR) [reply tests/integration_tests/test_hyperopt.py]: Done

File: tests/integration_tests/test_hyperopt.py (hunk 6 above)
Comment (ShreyaR) [reply tests/integration_tests/test_hyperopt.py]: Done

File: tests/integration_tests/test_hyperopt_ray.py (hunk 7 above)
Comment (ShreyaR) [reply tests/integration_tests/test_hyperopt_ray.py]: Done

File: tests/integration_tests/test_hyperopt_ray.py (hunk 8 above)
Comment (ShreyaR) [reply tests/integration_tests/test_hyperopt_ray.py]: Done

File: tests/integration_tests/test_hyperopt_ray_horovod.py (hunk 9 above)
Comment (ShreyaR) [reply tests/integration_tests/test_hyperopt_ray_horovod.py]: Done

File: tests/integration_tests/utils.py (hunk 10 above)
Comment (ShreyaR) [reply tests/integration_tests/utils.py]: Done

File: tests/ludwig/features/test_image_feature.py (hunk 11 above)
Comment (ShreyaR) [reply tests/ludwig/features/test_image_feature.py]: Done

**Comment by ShreyaR** [comment]
\\\\\\\\\\\\

**Pull request merged**